test/Rings/FreeAssociativeAlgebraIdeal.jl

```diff
@@ -114,3 +112,13 @@ end
   x = base_ring(ideal(gb5))[1]; y = base_ring(ideal(gb5))[7]
   @test x*y - y*x in ideal(gb5)
 end
+
+@testset "FreeAssociativeAlgebraIdeal.groebner_basis.f4" begin
+    S1 = quantum_symmetric_group(4);
+    S1_alt = quantum_symmetric_group(4);
+    gb_f4 = groebner_basis(S1;algorithm=:f4)
+    gb_bb = groebner_basis(S1_alt;algorithm=:buchberger);
+    @test is_groebner_basis(gb_f4)
+    @test is_groebner_basis(gb_bb)
+    S1 == S1_alt
```

Comment:
Do you maybe mean 
```suggestion
    gb_f4 == gb_bb
```
? Or is the equality of quantum symmetric groups depending on the groebner bases an object knows?

Reply:
The equality of the two ideals checks whether the generating sets  are contained in the other ideal. The Gröbner bases themselves are not necessarily the same.

Reply:
Ah okay